FILE .mise.toml
RANGE {"path":".mise.toml","old_lines":[0,0],"new_lines":[1,9],"diff_block_lines":[6,15],"hunk_header":"@@ -0,0 +1,9 @@","substantive_changed_lines":7}
@@ -0,0 +1,9 @@
+[env]
+ANDROID_HOME = "{{env.HOME}}/.local/share/mise/installs/android-sdk/1.0"
+_.path = [
+  "{{env.HOME}}/.local/share/mise/installs/android-sdk/1.0/platform-tools",
+  "{{env.HOME}}/.local/share/mise/installs/android-sdk/1.0/emulator",
+]
+
+[tools]
+java = "17"
